### PR TITLE
Update using-graphql.md

### PR DESCRIPTION
**Pull request by minaibrahimsaid**

### CODE REVIEW

File: docs/pages/guides/using-graphql.md [hunk 1]
```diff
@@ -324,7 +324,7 @@ class CreatePage extends React.Component {
             disabled={!this.state.text || !this.state.title}
             type="submit"
             value="Create"
-          />{' '}
```
Comment (cruzach) [inline docs/pages/guides/using-graphql.md:327]: this is actually necessary in some cases (https://stackoverflow.com/questions/38547558/why-do-people-put-in-their-react-jsx), can I ask why you want it removed?

Comment (minaibrahimsaid) [reply docs/pages/guides/using-graphql.md:327]: yes it's necessary in some cases  but it' not a good idea to add space like this you can do this 

```{` or cancel`}``` inside <a>

Comment (cruzach) [reply docs/pages/guides/using-graphql.md:327]: okay, could you change that in your PR?

Comment (minaibrahimsaid) [reply docs/pages/guides/using-graphql.md:327]: sure i can

**Review by cruzach — APPROVED** [review]

**Pull request merged**